### PR TITLE
Add documentation examples for `invalid-unary-operand-type`

### PR DESCRIPTION
**Pull request by julthep**
Co-authored-by: Vladyslav Krylasov <vladyslav.krylasov@gmail.com>

## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :scroll: Docs          |

## Description
Add documentation examples for `invalid-unary-operand-type`

Refs #5953

### CODE REVIEW

File: doc/data/messages/i/invalid-unary-operand-type/bad.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+fruit = int
+worm = - fruit  # [invalid-unary-operand-type]
```
Comment (Pierre-Sassoulas) [inline doc/data/messages/i/invalid-unary-operand-type/bad.py:2]:
```suggestion
cherries = 10
eaten_cherries = int
cherries = - eaten_cherries  # [invalid-unary-operand-type]
```

File: doc/data/messages/i/invalid-unary-operand-type/good.py [hunk 2]
```diff
@@ -1 +1,3 @@
-# This is a placeholder for correct code for this message.
+fruit = 1
+worm = 10
+worm -= fruit
```
Comment (Pierre-Sassoulas) [inline doc/data/messages/i/invalid-unary-operand-type/good.py:3]:
```suggestion
cherries = 10
eaten_cherries = 2
cherries -= eaten_cherries
```

**Review by Pierre-Sassoulas — CHANGES_REQUESTED** [review]
I think the example makes more sense that way, let me know what you think :)

**Review by Pierre-Sassoulas — APPROVED** [review]

File: doc/data/messages/i/invalid-unary-operand-type/bad.py (hunk 1 above)
Comment (julthep) [reply doc/data/messages/i/invalid-unary-operand-type/bad.py]: Yes, both examples make much more sense. :) Thank you very much.

File: doc/data/messages/i/invalid-unary-operand-type/good.py (hunk 2 above)
Comment (julthep) [reply doc/data/messages/i/invalid-unary-operand-type/good.py]: Yes, both examples make much more sense. :) Thank you very much.

**Pull request merged**